### DISCUSSION TITLE
Set simulator to verilator for cocotb-test

### DISCUSSION THREAD

**Discussion by TomO700** [cocotb/cocotb#2995] [Q&A] [answered]
Can I please know how the simulator can be changed to ventilator when using cocotb-test. 
I can find the method for the make file route but not for this. 

Thanks

**Comment by themperek** [comment, marked as answer]
not `ventilator`  but `verilator`. Using environmental variable `SIM=verilator`

**Reply by TomO700** [reply]
Sorry typo(auto correct)
